packages/shared/src/permissions/permissions.utils.ts

```diff
@@ -28,6 +25,45 @@ export function hasPermission(
   );
 }
 
+export const userHasPermission = (
+  superAdmin: boolean,
+  userPermissions: UserPermissions,
+  permission: Permission,
+  project?: string | string[],
+  envs?: string[]
+): boolean => {
+  if (superAdmin) {
+    return true;
+  }
+
+  let checkProjects: (string | undefined)[];
+  if (Array.isArray(project)) {
+    checkProjects = project.length > 0 ? project : [undefined];
+  } else {
+    checkProjects = [project];
+  }
+
+  if (["readData", "viewEvents", "runQueries"].includes(permission)) {
```

Comment:
Don't hard code this list here.  Put it in the organization.util file where the permissions are currently defined.  When we need to add a new permission, we don't want to have to remember to edit 2 different files in different packages.

Reply:
@jdorn That's where I had it originally, but in doing so I got a compiler error when trying to import from that file in the `shared` directory.